images/cilium/Dockerfile

```diff
@@ -83,7 +83,8 @@ ARG TARGETOS
 ARG TARGETARCH
 LABEL maintainer="maintainer@cilium.io"
 RUN echo ". /etc/profile.d/bash_completion.sh" >> /etc/bash.bashrc
-COPY --from=cilium-envoy / /
+COPY --from=cilium-envoy /usr/lib/libcilium.so /usr/lib/libcilium.so
```

Comment:
This can be removed as we are still building proxylib (e.g libcilium.so) from cilium/cilium repo in 1.12